src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), new Random()
+                .nextInt(MAX_RANDOM_NUMBER + 1));
+    }
```

Comment:
here we have same problem as we had in `ColorSupplier` class: 
we create new instance of `Random` class for every method run